### PR TITLE
adds larivaar assist functionality

### PR DESCRIPTION
**Pull request by inderpreetsingh**
Screenshot 2020-01-29 at 4 08 48 PM
Added larivaar assist functionality that overtakes vishraam styles when its on.

### CODE REVIEW

**Comment by maneetpaul** [comment]
Will this work on Chromecast and is it compatible across dark/light themes?

**Comment by inderpreetsingh** [comment]
@maneetpaul I have added theme support by using the theme respective yamki color.

**Comment by inderpreetsingh** [comment]
I will send chromecast PR on chromecast repo.

File: www/src/scss/themes/helpers/_theme-generator.scss [hunk 1]
```diff
@@ -45,6 +45,12 @@
         );
       }
     }
+
+    &.larivaar.larivaar-assist {
```
Comment (tsingh777) [inline www/src/scss/themes/helpers/_theme-generator.scss]: so larivaar assist will have the highest priority? would some messaging help so users don't get confused?

**Comment by maneetpaul** [comment]
This should work similar to our vishraam options. Where Larivaar assist is only an available option if Larivaar is first turned on. If Larivaar is turned off, then assist is also automatically turned off and disabled. Once assist is turned off, vishraams come back as their previous state. 

That should eliminate any confusion.

**Comment by ManjotS** [comment]
Is it possible to have vishraam supercede assist? So we can use both?

**Comment by maneetpaul** [comment]
@ManjotS, for now, we are going to have it be one or another. We'll enhance this down the road to allow for both. I also want to see what usage is like for larivaar assist in our analytics.

**Comment by ManjotS** [comment]
It will be low because:

- very few people know or care what larivaar is (as percentage of panth)
- Of those that know, a smaller portion care to use it
- Of those that know, some are against using it

**Comment by inderpreetsingh** [comment]
https://youtu.be/75D_jcbLp8s - Larivaar options :)

**Pull request merged**